Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/photo_field.tsx

```diff
@@ -330,9 +327,9 @@ export class ChangePictureModal extends
                 style={{ ...imageStyle, ...imageScaling }}/>
             </div>
             <Padding size={ChangePictureModal.PADDING_BETWEEN_ELEMENTS}/>
-            <Slider onChangeValue={this.onSliderMovement}
-              scaleValue={this.state.imageScaling}
-              isReadOnly={isSliderReadOnly}/>
+            <Slider onChange={this.onSliderMovement}
+              scale={this.state.scaling}
+              readonly={Boolean(!this.props.imageSource)}/>
```

Comment:
If you're doing ```!xxx``` then you don't need to call ```Boolean``` since the ```!``` operator will convert the parameter to a ```boolean``` automatically.